Header header
geometry_msgs/Point ground_truth
geometry_msgs/Point amcl_pose
float64 lon_error
float64 lat_error
float64 yaw_error
float64 abs_error
